app/src/testPlay/java/com/duckduckgo/app/referrer/AppReferrerInstallPixelSenderTest.kt

```diff
@@ -35,14 +35,16 @@ class AppReferrerInstallPixelSenderTest {
     private val pixel: Pixel = mock()
     private val appBuildConfig: AppBuildConfig = mock()
     private val appReferrerDataStore: AppReferrerDataStore = mock()
-    private val statisticsDataStore: StatisticsDataStore = mock()
     private val playStoreInstallChecker: VerificationCheckPlayStoreInstall = mock()
     private val captor = argumentCaptor<Map<String, String>>()
 
     @Before
     fun setup() {
         whenever(appBuildConfig.deviceLocale).thenReturn(Locale.US)
         whenever(playStoreInstallChecker.installedFromPlayStore()).thenReturn(true)
+        runBlocking {
```

Comment:
nit: you can remove `runBlocking` if you define the function as `fun setup() = runTest {  }`